api/bin/chainflip-cli/src/settings.rs

```diff
@@ -88,6 +88,16 @@ pub enum BrokerSubcommands {
 	/// Request a swap deposit address.
 	RequestSwapDepositAddress(SwapRequestParams),
 	WithdrawFees(WithdrawFeesParams),
+	/// Register this account as a broker account.
+	RegisterAccount,
+	/// De-register this broker account.
+	DeregisterAccount {
```

Comment:
The --force help doesn't show here for the validator subcommand, but it does for the lp one 🤔 
```
./target/debug/chainflip-cli validator deregister-account -h
chainflip-cli-validator-deregister-account 
De-register this validator account

USAGE:
    chainflip-cli validator deregister-account

OPTIONS:
    -h, --help    Print help information
```

Reply:
The validator command doesn't have the force option because validators don't own any assets.